DDG4/python/DDG4.py

```diff
@@ -424,7 +424,7 @@ def setupUI(self, typ='csh', vis=False, ui=True, macro=None):
     ui_action.SessionType = typ
     if macro:
       ui_action.SetupUI = macro
-    self.master().registerGlobalAction(ui_action)
+    self.master().registerGlobalAction(str(ui_action))
```

Comment:
```suggestion
    self.master().registerGlobalAction(ui_action)
```
That made things worse